components/apimgt/org.wso2.carbon.apimgt.throttling.siddhi.extension/pom.xml

```diff
@@ -33,7 +33,7 @@
         <repository>
             <id>wso2</id>
             <name>WSO2 Repository</name>
-            <url>http://maven.wso2.org/nexus/content/groups/wso2-public/</url>
+            <url>https://maven.wso2.org/nexus/content/groups/wso2-public/</url>
```

Comment:
can't we just remove this repositories section since this already in parent pom

Reply:
Removed in https://github.com/wso2/carbon-apimgt/pull/10893/commits/1f50e8b0cff88eb18f13060bc7c878910c1bdfd0